Library/Formula/capnp.rb

```diff
@@ -14,6 +14,7 @@ class Capnp < Formula
   option "without-shared", "Disable building shared library variant"
 
   def install
+    ENV.cxx11 if MacOS.version < :mavericks
```

Comment:
We've already added `needs :cxx11` to the formula previously in https://github.com/Homebrew/homebrew/commit/15ca054a15eff279232672d3d2b59abbae148fb6 - Just missed the `ENV` apparently.